### DISCUSSION TITLE
Urgent Assistance Required for GitHub Account help please

### DISCUSSION THREAD

**Discussion by sukrucildirr** [community/community#135877] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

account @Budalebah flagged , this account has multiple contributions in projects and now account down. this situatin effects many projects. please help to un flag it. after a project started to a workflow accoount flagged. this is an urgent call.
this account greatly value the services provided by GitHub and contrituting to many projects. If there are any additional details or documentation required to support please let me know, and I will be happy to provide them.

Thank you for your attention to this matter. I look forward to your prompt response.

**Comment by ebndev** [comment]
Hi @sukrucildirr,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
